cdn/agent-loader/lite.js

```diff
@@ -2,16 +2,13 @@
  * Copyright 2020 New Relic Corporation. All rights reserved.
  * SPDX-License-Identifier: Apache-2.0
  */
-import { detectPolyfillFeatures } from './utils/feature-detection'
-const PolyfillFeatures = detectPolyfillFeatures();
```

Comment:
Why were these lines removed?

Reply:
these are used to report supportability metrics

Reply:
Oh